### ISSUE TITLE
[DynamicSideContent]:  aria-label for the side panel is meaningless and not customizable and missing  aria-label  for the main conent

### ISSUE THREAD

**Issue by yzy415** [SAP/ui5-webcomponents#8612]
### Bug Description

As shown below, the aria-label for the side panel is "Side Content" which is too generic and not customizable. We should support accepting the consumer-defined aria-lable for the side panel.

Besides, the aria-label for the main content is missed.

![image](https://github.com/SAP/ui5-webcomponents/assets/53516646/6e82a825-61c3-4344-90a2-79dceb1656f5)

### Affected Component

DynamicSideContent

### Expected Behaviour

aria-label for the side panel should be customizable and we should also give the main content a customizable aria-label.

### Steps to Reproduce

1. go to ui5 [DynamicSideContent documentation](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicsidecontent--docs) 
2. inspect the side content and main content

### Priority

None

### UI5 Web Components Version

1.26.1

### Browser

Chrome, Edge, Firefox, Safari

### Organization

SAP SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

There is a feature request to provide an API to set custom labels on the side content area/contianer. 
Also, the main area/container seems to lack accessibility attributes (like role and aria-label). 
Could you please have a look?

Kind Regards,
Dobrin

**Comment by NHristov-sap** [comment]
Hi @yzy415 ,

The request is now internally logged as FIORITECHE1-8187, and we'll promptly notify you once it's been addressed.
Thanks & Regards,

Nikolay Hristov
